app/controllers/contact_controller.rb

```diff
@@ -0,0 +1,23 @@
+class ContactController < ApplicationController
+  def new
+    @contact_form = ContactForm.new
+  end
+
+  def create
+    @contact_form = ContactForm.new
+
+    if @contact_form.submit(form_params)
+      UserMailer.contact_request(form_params).deliver_later
+      flash[:success] = t('contact.messages.thanks')
+      redirect_to contact_url
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def form_params
+    params.require(:contact_form).permit(:want_learn, :want_tell, :email_or_tel, :comments)
+  end
+end
```

Comment:
Can you write a controller test for this, please? It should verify:
- [ ] the redirect
- [ ] the flash message
- [ ] that the expected mailer was invoked
- [ ] that it renders the `:new` template when the form is invalid

 You can look at this similar one to test the mailer: https://github.com/18F/identity-idp/blob/master/spec/forms/update_user_email_form_spec.rb#L26-L29